ui/pages/swaps/build-quote/build-quote.js

```diff
@@ -348,6 +365,7 @@ export default function BuildQuote({
 
   useEffect(() => {
     dispatch(resetSwapsPostFetchState());
+    dispatch(setReviewSwapClickedTimestamp());
```

Comment:
Is this dispatch just to set the reviewSwapClickedTimestamp to`undefined`?

Reply:
Exactly that. When the Build Quote page is loaded, I want to clean it up and set a fresh timestamp on submit.